src/main/java/hudson/plugins/sonar/SonarPublisher.java

```diff
@@ -260,6 +265,10 @@ public void setBranch(final String branch) {
     this.branch = branch;
   }
 
+  /**
+   * @deprecated since 2.2
+   */
+  @Deprecated
   public String getLanguage() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.internal.sonarsource.com/coding_rules#rule_key=squid%3AS1133)